pkg/controller/aws.go

```diff
@@ -73,12 +73,15 @@ import (
 	"github.com/crossplane-contrib/provider-aws/pkg/controller/sns"
 	"github.com/crossplane-contrib/provider-aws/pkg/controller/sqs"
 	"github.com/crossplane-contrib/provider-aws/pkg/controller/transfer"
+	"github.com/crossplane-contrib/provider-aws/pkg/utils/metrics"
 	"github.com/crossplane-contrib/provider-aws/pkg/utils/setup"
 )
 
 // Setup creates all AWS controllers with the supplied logger and adds them to
 // the supplied manager.
 func Setup(mgr ctrl.Manager, o controller.Options) error {
+	metrics.SetupMetrics(mgr, o)
```

Comment:
I wonder if it would be better to call this in `cmd/provider/main.go` since this is some sort of "meta" feature that is not directly related to the controller setup.

Reply:
Agreed, I will make these changes